data/available_versions.go

```diff
@@ -42,44 +42,47 @@ func NewAvailableVersionsFromAPI(
 		rwm:             new(sync.RWMutex),
 		cache:           nil,
 		baseVersionsURL: baseVersionsURL,
-		restClient:      restCl,
+		apiClient:       apiClient,
 	}
 }
 
 // Refresh method for AvailableVersionsFromAPI
-func (a availableVersionsFromAPI) Refresh(cluster types.Cluster) ([]types.ComponentVersion, error) {
-	reqBody := SparseComponentAndTrainInfoJSONWrapper{}
+func (a availableVersionsFromAPI) Refresh(cluster models.Cluster) ([]models.ComponentVersion, error) {
+	reqBody := operations.GetComponentsByLatestReleaseBody{}
 	for _, component := range cluster.Components {
-		sparseComponentAndTrainInfo := SparseComponentAndTrainInfo{}
-		sparseComponentAndTrainInfo.Component.Name = component.Component.Name
-		sparseComponentAndTrainInfo.Version.Train = component.Version.Train
-		reqBody.Data = append(reqBody.Data, sparseComponentAndTrainInfo)
-	}
-	js, err := json.Marshal(reqBody)
-	if err != nil {
-		return []types.ComponentVersion{}, err
+		cv := new(models.ComponentVersion)
+		cv.Component = &models.Component{}
+		cv.Version = &models.Version{}
+		cv.Component.Name = component.Component.Name
+		if strings.Contains(component.Version.Version, "stable") {
+			cv.Version.Train = "stable"
+		} else {
+			cv.Version.Train = "beta"
```

Comment:
currently there is no alpha version release and this need to be handled in a different way than getting from the version....https://github.com/deis/workflow-manager/issues/61